### PR TITLE
Feature/Create Home page from designs

### PR DESCRIPTION
**Pull request by will-nemo**

### CODE REVIEW

**Comment by will-nemo** [comment]
Still working on writing tests

File: src/style-sheets/generic/_button.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.button {
+    width: 189px;
+    height: 49px;
+    border-radius: 4px;
+    border-width: 0;
+
+    font-size: 15pt;
```
Comment (Fabricevladimir) [inline src/style-sheets/generic/_button.scss]: I noticed previously you were using `em`. Is there a reason for the change now?

File: src/style-sheets/modules/_colors.scss [hunk 2]
```diff
@@ -3,6 +3,13 @@ $olGreen: #296c37;
...
+//Brand Themes
+$olLTPrimary: #FFFFFF;
+$olLTAccent: #43A850;
+
+$olLTHeader: #262626;
+$olLTCopy: #2B2D31;
+
 // Supporting Colors
 $white: #ffffff;
 $black: #000000;
```
Comment (Fabricevladimir) [inline src/style-sheets/modules/_colors.scss]: I like the variable for all the colors. Could we do that with font-size as well?

Comment (will-nemo) [reply src/style-sheets/modules/_colors.scss]: Yup

File: src/style-sheets/generic/_button.scss (hunk 1 above)
Comment (will-nemo) [reply src/style-sheets/generic/_button.scss]: Fixed it

File: src/style-sheets/modules/_colors.scss (hunk 2 above)
Comment (will-nemo) [reply src/style-sheets/modules/_colors.scss]: Done

**Comment by will-nemo** [comment]
#103

File: src/style-sheets/modules/_font.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+//Font Sizes
```
Comment (Fabricevladimir) [inline src/style-sheets/modules/_font.scss]: Yay sizes!

File: src/components/FeatureContainer/FeatureContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+/**
+* Component that displays a container with text children passed in aligned opposite from an image
+* Takes in a {@class JSX} image and a (@class boolean) invert, that indicates with side to position the text/
```
[the remark this answers is not in the document]
Comment (will-nemo) [reply src/components/FeatureContainer/FeatureContainer.jsx]: I'd be down to have a conversation as setting prototypes as a standard.

**Pull request merged**